deployments/arbitrum/usdc/migrations/1679020486_configurate_and_ens.ts

```diff
@@ -0,0 +1,355 @@
+  l1DeploymentManager: DeploymentManager,
+  l2DeploymentManager: DeploymentManager
+) {
+  const { l1GatewayRouter } = await l1DeploymentManager.getContracts();
+
+  const l1GatewayAddress = await l1GatewayRouter.getGateway(token);
+  const l1Gateway = new Contract(
+    l1GatewayAddress,
+    gatewayInterface,
+    l1DeploymentManager.hre.ethers.provider
+  );
+  const l2GatewayAddress = await l1Gateway.counterpartGateway();
+
+  const data = gatewayInterface.encodeFunctionData(
+    'finalizeInboundTransfer',
+    [
+      token,  // address _token,
+      from,   // address _from,
+      to,     // address _to,
+      amount, // uint256 _amount,
+      utils.defaultAbiCoder.encode(
+        ['bytes', 'bytes'], ['0x', '0x']
+      ) // bytes calldata _data
+    ]
+  );
+
+  return await estimateL2Transaction(
+    {
+      from: applyL1ToL2Alias(l1GatewayAddress),
+      to: l2GatewayAddress,
+      data
+    },
+    l2DeploymentManager
+  );
+}
+
+export default migration('1679020486_configurate_and_ens', {
+  prepare: async (_deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const {
+      bridgeReceiver,
+      comet,
+      cometAdmin,
+      configurator,
+      rewards,
+    } = await deploymentManager.getContracts();
+
+    const {
+      inbox,
+      l1GatewayRouter,
+      timelock,
+      governor,
+      USDC,
+      COMP,
+    } = await govDeploymentManager.getContracts();
+
+    const USDCAmountToBridge = exp(10_000, 6);
+    const COMPAmountToBridge = exp(2_500, 18);
+    const usdcGatewayAddress = await l1GatewayRouter.getGateway(USDC.address);
+    const compGatewayAddress = await l1GatewayRouter.getGateway(COMP.address);
+    const refundAddress = comet.address; // XXX timelock address?
+
+    const compGasParams = await estimateTokenBridge(
+      {
+        token: COMP.address,
+        from: timelock.address,
+        to: rewards.address,
+        amount: COMPAmountToBridge
+      },
+      govDeploymentManager,
+      deploymentManager
+    );
+
+    const usdcGasParams = await estimateTokenBridge(
+      {
+        token: USDC.address,
+        from: timelock.address,
+        to: comet.address,
+        amount: USDCAmountToBridge
+      },
+      govDeploymentManager,
+      deploymentManager
+    );
+
+    const configuration = await getConfigurationStruct(deploymentManager);
+
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(comet.address, configuration)
+    );
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
```

Comment:
We also need to `setRewardConfig` in the rewards contract. See this example https://github.com/compound-finance/comet/blob/c91bffce984ec6f8276b6f7e9ef362d9293416c1/deployments/optimism-goerli/usdc/migrations/1679592519_configurate_and_ens.ts#L77

Reply:
good call. updated